draft-bormann-asdf-instance-information.md

```diff
@@ -211,16 +215,74 @@ Non-affordance:
 
 # Instance Information and SDF
 
-Instantiation doesn't produce an instance (ouch), which is the device,
-twin, etc., but a message.
+The instantiation of an SDF model does not produce an instance, which is, for example, a physical device or a digital twin.
+Instead, the instantiation produces an instance-related _message_, which adheres to a uniform message format and is always controlled by the corresponding SDF model.
+Depending on the recipient and the purpose of the message, different fields of the message format are present, reporting different kinds of information related to a Thing or causing it to change the state of the Thing when consumed by the recipient.
+Taking into account previous revisions of this document as well as {{-non-affordance}}, the potential use cases for instance-related messages indicate the following archetypes:
+
+1. *Context snapshots* that only report context information about a Thing
+2. *Proofshots* that report a Thing's state (or parts of it), which may include context information
+3. *Delta messages* that indicate state changes compared to a previous context snapshot or proofshot message
+4. *Patch messages* that cause a Thing's state to change
+5. *Identity manifests* that report information related to a Thing's identity
```

Comment:
Maybe we should collect all subset/variant proofshots under that headline

Reply:
You mean identity manifest?